lib/vdsm/storage/mpathconf.py

```diff
@@ -130,3 +174,33 @@ def read_blacklist():
         wwids.add(wwid)
 
     return wwids
+
+
+def read_metadata():
+    """
+    The multipath configuration file at /etc/multipath.conf should contain
+    a tag in form "RHEV REVISION X.Y" for this check to succeed.
+    If the tag above is followed by tag "RHEV PRIVATE" the configuration
+    should be preserved at all cost.
+
+    Returns:
+        Metadata: Return a tuple where the first element is the revision
+            check result, and the second is private check result.
+    """
+    first = second = ''
+    with open(CONF_FILE) as f:
+        mpathconf = [x.strip("\n") for x in f.readlines()]
+    try:
+        first = mpathconf[0]
+        second = mpathconf[1]
+    except IndexError:
+        pass
+    private = _PRIVATE_TAG in second or _OLD_PRIVATE_TAG in second
```

Comment:
If we add:

    _PRIVATE_TAGS = ("# VDSM PRIVATE", "# RHEV PRIVATE")

We can use:

    private = second.startswith(_PRIVATE_TAGS)

Which is more correct and consistent.

Reply:
It looks really nice now that we use the same strategy in all cases, and all conditions have been simplified to use `startswith` :)